### PR TITLE
Add a top level page document our Kubernetes choices

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: content/embedded-kubernetes/_index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                    <p>The Replicated Embedded Kubernetes cluster is installed with Rook and Ceph, and the installer manages the state of the system. </p>
+                    <div class="u-marginTop--small">
+                        <a href="/guides/native-scheduler" class="Button secondary">Get Started</a>
+                    </div>
+                </div>
+                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
+                    <ul class="pros-list">
+                        <li class="u-commandPromptRed">
+                            <h6>Highly available storage</h6>
+                            <p>Rook provisions and operates a Ceph cluster to manage storage, a battle tested, mature storage system.</p>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]:
```suggestion
                            <p>Rook provisions and operates Ceph, a battle tested, mature system for distributed fault-tolerant storage.</p>
```

File: content/embedded-kubernetes/_index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                        </li>
+                        <li class="u-dockerRed">
+                            <h6>Relocatable Volumes</h6>
+                            <p>Automatic support for persistent volumes, which enables Kubernetes to reschedule Pods anywhere in the cluster, and the data will stioll be available.</p>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]: typo

```suggestion
                            <p>Automatic support for persistent volumes, which enables Kubernetes to reschedule Pods anywhere in the cluster while ensuring data moves with your workloads.</p>
```

File: content/embedded-kubernetes/_index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                    <p>Every Replicated Embedded Kubernetes cluster comes with Weave overlay networking configured and CoreDNS.</p>
+                    <div class="u-marginTop--small">
+                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
+                    </div>
+                </div>
+                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
+                    <ul class="pros-list">
+                        <li class="u-commandPromptBlue">
+                            <h6>CoreDNS Support</h6>
+                            <p>By default, Kubernetes will register service and pods with CoreDNS, allowing for container to container communication on the overlay network.</p>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]:
```suggestion
                            <p>By default, Kubernetes will register service and pods with CoreDNS, allowing for state-of-the art service discovery via DNS lookups.</p>
```

File: content/embedded-kubernetes/_index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+</div>
+
+<div class="scheduler-type main-section u-borderBottom--gray">
+    <div class="container">
+        <div class="paddingContainer">
+            <div class='u-flexTabletReflow'>
+                <div class='flex1 flex-column flex-verticalCenter u-lineHeight--normal left-content'>
+                    <span class="logo dockerSwarmLogo-combo"></span>
+                    <h5>Weave Networking</h5>
+                    <p>Every Replicated Embedded Kubernetes cluster comes with Weave overlay networking configured and CoreDNS.</p>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]:
```suggestion
                    <p>Every Replicated Embedded Kubernetes cluster comes with secure-by-default Weave overlay networking for inter-container communication</p>
```

File: content/embedded-kubernetes/_index.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                    </div>
+                </div>
+                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
+                    <ul class="pros-list">
+                        <li class="u-commandPromptBlue">
+                            <h6>CoreDNS Support</h6>
+                            <p>By default, Kubernetes will register service and pods with CoreDNS, allowing for container to container communication on the overlay network.</p>
+                        </li>
+                        <li class="u-ymlBlueLight">
+                            <h6>Easier On Firewalls</h6>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]: Don't have an alternative, but this heading feels a little off tone-wise

File: content/embedded-kubernetes/_index.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+                    <p>Contour is a portable ingress controller from Heptio.</p>
+                    <div class="u-marginTop--small">
+                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
+                    </div>
+                </div>
+                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
+                    <ul class="pros-list">
+                        <li class="u-commandPromptBlue">
+                            <h6>Portable</h6>
+                            <p>The contour ingress controller is compatible anywhere the cluster is installed; cloud providers and care metal incldued.</p>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]:
```suggestion
                            <p>The contour ingress controller is compatible anywhere the cluster is installed; cloud providers and bare metal incldued.</p>
```

File: content/embedded-kubernetes/_index.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+                        </li>
```
Comment (dexhorthy) [inline content/embedded-kubernetes/_index.md]: I'd add another few headers here:

- Built on Envoy
    - Built on the mature, battle-tested Envoy proxy 
- Cloud native, on prem
    - Instant reconfiguration, dynamic ingress routes, and native WebSocket support
- The future of Ingress
    - Leverage next-generation Ingress features like route inheritance and upstream health checking

File: content/embedded-kubernetes/_index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+                    <p>Contour is a portable ingress controller from Heptio.</p>
+                    <div class="u-marginTop--small">
+                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
+                    </div>
+                </div>
+                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
+                    <ul class="pros-list">
+                        <li class="u-envoyLogoBlue">
+                            <h6>Built On Envoy</h6>
+                            <p>Built on the mature, battle-tested Envoy proxy that powers dynamic infrastructure at massive scale.</p>
```
Comment (divolgin) [inline content/embedded-kubernetes/_index.md]: "mature battle tested" twice on the same page, makes it look like just buzz words

**Pull request merged**